components/sup/tests/utils/mod.rs

```diff
@@ -120,3 +125,46 @@ where
         }
     }
 }
+
+fn write_default_svc_user_and_group_metafiles<P>(hab_pkg_path: P)
```

Comment:
Maybe add the detail from the PR message as a docstring on this function?

Reply:
Good call!